rxandroid/src/main/AndroidManifest.xml

```diff
@@ -12,4 +12,4 @@
   See the License for the specific language governing permissions and
   limitations under the License.
 -->
-<manifest package="rx.android" />
\ No newline at end of file
+<manifest package="io.reactivex.android" />
```

Comment:
hm, that's weird, I was under impression that both group name of the artifact and package name of classes contain `2`